dbConnect/dbConnect.py

```diff
@@ -43,14 +44,16 @@ def connect(self):
                 raise ValueError(err)
         self.cursor = self.connection.cursor()
 
-    def __init__(self, credentials_file='credentials.json', host=None, user=None, password=None, database=None):
+    def __init__(self, credentials_file='credentials.json', host=None, user=None, password=None, database=None, port=3306):
         """
         Initialise object with credentials file provided
         """
         if host and user and password and database:
-            self.settings = {"host": host, "user": user, "password": password, "database": database}
+            self.settings = {"host": host, "user": user, "password": password, "database": database, "port": port}
         else:
             self.settings = json.load(open(credentials_file, 'r'))
+            if 'port' not in self.settings:
+                self.settings['port'] = 3306
```

Comment:
make it equal to port variable. We already initialised port parameter with default value 3306. ``self.settings['port'] = port``